lib/HLSL/DxilPreparePasses.cpp

```diff
@@ -838,6 +915,11 @@ class DxilFinalizeModule : public ModulePass {
         convertQuadVote(M, DM.GetOP());
       }
 
+      // Convert IsSpecialFloat intrinsics
```

Comment:
Wonder if this would be a good place to document why we're doing this?  Someone reading this without the context we have today would be pretty confused.